### ISSUE TITLE
String.to_existing_atom failing in utils.ex

### ISSUE THREAD

**Issue by njwest** [ExHammer/hammer#88]
**Describe the bug**
When calling Hammer functions with a single backend, NOT a list of backends, the call fails with:

```
** (exit) an exception was raised:
    ** (ArgumentError) errors were found at the given arguments:

  * 1st argument: not an already existing atom

        :erlang.binary_to_existing_atom("hammer_backend_ets_pool", :utf8)
        (hammer 6.2.0) lib/hammer/utils.ex:9: Hammer.Utils.pool_name/1
        (hammer 6.2.0) lib/hammer.ex:255: Hammer.call_backend/3
        (hammer 6.2.0) lib/hammer.ex:51: Hammer.check_rate/4
```

** Provide the following details

- Elixir version (elixir -v): Elixir 1.15.6 
- Erlang version (erl -v):  Erlang/OTP 26 
- Operating system: macOS 14.2 

**Expected behavior**
I expect `Hammer.check_rate/4` to not result in an error

**Actual behavior**
See above

**Comment by njwest** [comment]
@epinault is there a reason why this needs to be `to_existing_atom/1`? I can throw up a quick PR similar to #87 that just changes this to `to_atom/1` but I don't wanna do that if there is an ideological resistance to `to_atom/1`

**Comment by epinault** [comment]
I think it s to avoid some risk of Atom during configuration but we can make it to_atom for now  if you open an MR

**Comment by njwest** [comment]
> I think it s to avoid some risk of Atom during configuration but we can make it to_atom for now if you open an MR
> 
> https://hexdocs.pm/credo/Credo.Check.Warning.UnsafeToAtom.html

@epinault a fix for this has been added to PR #87

**Comment by wsbinette** [comment]
+1 also need this, can't deploy new project without it. 🙏🏼

**Comment by epinault** [comment]
Hello, I am have released a new version 6.2.1. I am unfortunately traveling for another few days and have limited access so hopefully that will unblock you

**Comment by njwest** [comment]
Thank you @epinault !!! 🙏 that solved it for me
